packages/admin-ui/src/page/header.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalHeading as Heading,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+
+export default function Header( {
+	breadcrumbs,
+	badges,
+	title,
+	subTitle,
+	actions,
+}: {
+	breadcrumbs?: React.ReactNode;
+	badges?: React.ReactNode;
+	title?: React.ReactNode;
+	subTitle: React.ReactNode;
+	actions?: React.ReactNode;
+	showSidebarToggle?: boolean;
+} ) {
+	return (
+		<VStack className="admin-ui-page__header" as="header">
+			<HStack
+				className="admin-ui-page__header-title"
+				justify="space-between"
+				spacing={ 2 }
+			>
+				<HStack spacing={ 2 }>
+					{ title && (
+						<Heading as="h2" level={ 3 } weight={ 500 } truncate>
+							{ title }
+						</Heading>
+					) }
+					{ breadcrumbs }
```

Comment:
Breadcrumbs sounds like a very handy feature as we think about admin pages with potentially deep structures like settings. How does this render? Should breadcrumbs appear _before_ the title, with the title acting as the 'current item' in the trail?